src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/LanguageClient/DefaultRazorLanguageClientMiddleLayer.cs

```diff
@@ -9,7 +9,7 @@
 namespace Microsoft.VisualStudio.Razor.LanguageClient;
 
 [Export(typeof(RazorLanguageClientMiddleLayer))]
-internal class DefaultRazorLanguageClientMiddleLayer : RazorLanguageClientMiddleLayer
+internal class DefaultRazorLanguageClientMiddleLayer : RazorLanguageClientMiddleLayer //, ILanguageClientMiddleLayer2<JToken>
```

Comment:
?

Reply:
Can this be deleted instead of commented out? At the very least, could a comment be added so that we're not scratching our heads about this later?